vantage6/server/resource/vpn.py

```diff
@@ -86,6 +93,68 @@ def get(self):
 
         return {'ovpn_config': ovpn_config}, HTTPStatus.OK
 
+    @with_node
+    def post(self):
+        """ Update an OVPN configuration file
+        ---
+        description: >-
+            Returns an OVPN configuration file with renewed keypair. This
+            allows the node to connect to EduVPN when the keypair was
+            invalidated. To access this endpoint, the node needs to be
+            authenticated.\n\n
+
+            Accesible for: `node`\n\n
+
+        responses:
+            200:
+                description: Ok
+            401:
+                description: Unauthorized or missing permission
+            501:
+                description: The vantage6-server has no VPN service
+            503:
+                description: The VPN server is not reachable
+
+        security:
+            - bearerAuth: []
+
+        tags: ["Other"]
+
+        """
+        # check if the VPN server is configured
+        if not self._is_server_configured():
+            return {'msg': 'This server does not support VPN'}, \
+                HTTPStatus.NOT_IMPLEMENTED
+
+        # retrieve user based on email or username
+        body = request.get_json()
+        vpn_config = body.get("vpn_config")
+        if not vpn_config:
+            return {"msg": "vpn_config is missing!"}, \
+                HTTPStatus.BAD_REQUEST
```

Comment:
Deze `400` mist in de docsting hierboven

Reply:
Added